pombola/za_hansard/management/commands/za_hansard_q_and_a_scraper.py

```diff
@@ -344,8 +345,26 @@ def handle_api_question_and_reply(self, data):
             if data[api_key]:
                 number_found = True
                 number_q_kwargs[filter_key] = data[api_key]
-        existing_kwargs = {'date__year': year, 'house': house}
+
+        try:
+            question_date = datetime.strptime(data['date'], "%Y-%m-%d").date()
+        except Exception:
+            self.stderr.write(
+                'Could not parse the date for question {}. Skipping.'.\
```

Comment:
I suspect that these failures won't be seen til we run it manually.
If we re-raise an exception with our better message, will that end up in the source parsing log we recently added?

We won't get mailed about this unless we let it exit with nonzero status - I think that's a bit overkill, but if it's easy to see this in admin, that'd be nice.

Reply:
You're right, we won't see the errors unless we exit with a non-zero status. It won't end up in the source parsing log either because this scripts fetches questions directly from the PMG questions API instead of parsing source documents. I could add something like a questions parsing log model?

Reply:
yeah let's do that and make it visible in Admin please. If it's easy, also send an error mail at the end if there are new failures logged - not sure if it's easy to identify errors seen before - let's timebox this to half a day? if it's gonna take more time than that I don't think it's worth it right now